examples/toolchain_v1alpha1_usersignup_cr.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: toolchain.dev.openshift.com/v1alpha1
-kind: UserProvisionRequest
+kind: UserSignup
```

Comment:
In this CR, I see - targetCluster: east-2a
So targetCluster will point to cluster_name and NOT cluster_url .. right ??
I am asking this just to make sure that these things are discussed ??

The only thing I see is, if we use cluster_url then we can directly use it.  If we use cluster_name then we need to query cluster_url (somehow) and then use cluster_url.  I generally found that we need cluster_url more than cluster_name.